streams/src/test/java/org/apache/kafka/streams/integration/AbstractResetIntegrationTest.java

```diff
@@ -106,10 +107,19 @@ private void prepareEnvironment() {
             kafkaAdminClient = (KafkaAdminClient) org.apache.kafka.clients.admin.AdminClient.create(commonClientConfig);
         }
 
+        try {
```

Comment:
Should we keep retrying until it succeed than just retry one more time? Is it possible that the second setCurrentTime fails again?

Reply:
If it falls into an endless loop, it means that we've discovered some other bugs. In Jenkins jobs we have a timeout so it will eventually timed out after that and people can be notified.

Reply:
Fair enough will update.